pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -63,6 +65,31 @@ private ElasticSearchParticipantDto(ElasticSearchParticipantDto.Builder builder)
 
     protected ElasticSearchParticipantDto() {  }
 
+    /**
+     * Changes the value for the given question's answer.
+     * Does not make any modification to underlying elastic data.
+     */
+    @VisibleForTesting
+    public void changeQuestionAnswer(String activityCode, String questionStableId, String value) {
+        boolean changedIt = false;
+        for (Activities activity : getActivities()) {
+            activity.getQuestionsAnswers();
```

Comment:
Is this line necessary? ("activity.getQuestionsAnswers();")

Reply:
Good catch!